ggshield/cmd/hmsl/fingerprint.py

```diff
@@ -47,7 +47,7 @@ def fingerprint_cmd(
     **_: Any,
 ) -> int:
     """
-    Collect secrets and prepare them to be queried.
+    Collect secrets and compute fingerprints.
```

Comment:
```suggestion
    Collect secrets and compute fingerprints to be used later by the `decrypt` command.
```

Reply:
The problem is that click will trim the sentence to 80 chars and replace with three dots any leftovers. Doing so will not improve a lot the message when doing `ggshield hmsl -h` and may make it worse.

Reply:
Maybe we could separate in two parts? The initial string to keep for `ggshield hmsl -h` and a paragraph below for more help that will be displayed when doing `ggshield hmsl fingerprint -h`.

click does support increasing the max_content_width but I don't think it's something we want to do.

Reply:
Indeed, I did so by using two lines :)